src/sysctl.rs

```diff
@@ -114,7 +115,7 @@ impl<T: SysctlStorage> Sysctl<T> {
         }
 
         let storage = Box::new(storage);
-        let mut table = vec![
+        let mut table = alloc::vec![
```

Comment:
I think `vec!` is traditional (it's in the standard prelude) so we should `use alloc::vec` here. (Eventually maybe define a prelude and do a wildcard import of it.)

Reply:
I'm not confident `use alloc::vec` will work because there's also a `mod` named `alloc::vec` 😬